File: W3_T3/person.ts
class Person {
    private age: number;

    constructor(public first_name: string, public last_name: string, age: number) {
        this.age = age;
    }

    getAge(): number {
        return this.age;
    }

    ageUp(): void {
        this.age++;
    }

    getFullname(): string {
        return `${this.first_name} ${this.last_name}`;
    }

    printFullname(): void {
        console.log(`${this.first_name} ${this.last_name}`);
    }
}

export default Person;

File: W3_T6/person.dart
class Person {
  late int _age;
  final String first_name;
  final String last_name;

  Person(this.first_name, this.last_name, int age) : _age = age;

  int getAge() {
    return _age;
  }

  void ageUp() {
    _age++;
  }

  String getFullname() {
    return '$first_name $last_name';
  }

  void printFullname() {
    print('$first_name $last_name');
  }
}